app/models/post_mover.rb

```diff
@@ -461,7 +461,9 @@ def copy_shifted_post_timings_to_temp
 
     # copy post_timings for shifted posts to a temp table using the new_post_number
     # they'll be copied back after delete_invalid_post_timings makes room for them
-    DB.exec <<~SQL
+    params = { post_ids: @post_ids_after_move }
```

Comment:
nit, but I don't think it's necessary to store this in a variable, this is fine:

```ruby
DB.exec(<<~SQL, post_ids: @post_ids_after_move)
```